addons/sms/wizard/sms_composer.py

```diff
@@ -127,7 +127,7 @@ def _compute_recipient_single(self):
                 continue
             records.ensure_one()
             res = records._sms_get_recipients_info(force_field=composer.number_field_name, partner_fallback=False)
-            composer.recipient_single_description = res[records.id]['partner'].name or records.display_name
+            composer.recipient_single_description = res[records.id]['partner'].name or records._sms_get_default_partners().display_name
```

Comment:
Note that this can technically be Falsy, though maybe not right now. 

But I guess it's fine to leave the field empty in those cases, it's only used for display purposes so that's fine.